Xamarin.Essentials/Maps/Map.android.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Globalization;
+using System.Threading.Tasks;
+using Android.Content;
+using AndroidUri = Android.Net.Uri;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Map
+    {
+        internal static Task PlatformOpenMapsAsync(double latitude, double longitude, MapLaunchOptions options)
+        {
+            var uri = string.Empty;
+            if (string.IsNullOrWhiteSpace(options.Name))
```

Comment:
Will this launch google maps if it is on the phone?

Reply:
I used to have to do some weird hackery to use the url.

Reply:
Yes it does